tests/integration/test_numeric_array.py

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
+
+"""Integration-test suite for numeric-arrays."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+
+from cr.cube.cube import Cube
+
+from ..fixtures import NA
+
+
+# pylint: disable=no-self-use
+class TestNumericArrays:
+    """Test-suite for numeric-arrays behaviors."""
+
+    def test_num_arr_x_cat_with_missings(self):
+        """Test means on numeric array, grouped by single categorical dimension."""
+        slice_ = Cube(NA.NUM_ARR_X_CAT_WITH_MISSINGS).partitions[0]
+
+        np.testing.assert_almost_equal(
+            slice_.means, [[87.66666667, 93.33333333, 1.0], [52.5, 50.0, 45.0]]
```

Comment:
When specifying the expectation with literal values like this, maybe use the same trick we used in the `cr.server` test, where you specified for each entry what it relates to. So here, it would be something like:
```
[
    #  Gender:
    #  M, F, Missing
    [87, 93, 1.0],  # Num Arr subvar: 0001
    [52, 50, 45],  # Num Arr subvar: 0002
]
```

Reply:
Same for other tests